.github/workflows/test-application.yaml

```diff
@@ -56,5 +52,5 @@ jobs:
             - name: Coverage
               if: ${{ matrix.coverage }}
               run: |
-                  wget https://scrutinizer-ci.com/ocular.phar
```

Comment:
This is necessary because the `ocular.phar` is not compatible with ^php8.0 see https://github.com/scrutinizer-ci/ocular/issues/54#issuecomment-784993370